src/moving_platform.hpp

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file moving_platform.hpp
+ * @brief     Moving platform class
+ * @author    Olivier verwoerd, Maarten Wassenaar, Daniel van Vliet & Jasper Smienk.
+ * @license   See LICENSE
+ */
+#include <stdint.h>

```

Comment:
hwlib provides int16_t types as well. If this module is compiled for Arduino it is advised to just include hwlib.